packages/core/src/plugins/manifest.ts

```diff
@@ -15,7 +15,7 @@ type ManifestByEntry = {
   };
   /** other assets (e.g. png、svg、source map) related to the current entry */
   assets: FilePath[];
-  html?: FilePath;
+  html?: FilePath[];
```

Comment:
For `initial`, `async` and `assets` , their value is an empty array when the file not exist.

Should `html` be consistent with other files?